src/dials/array_family/flex_ext.py

```diff
@@ -1297,14 +1299,44 @@ def map_centroids_to_reciprocal_space(
         # reciprocal lattice viewer -> but if we are looking at the crystal
         # coordinate frame we need to look at the experiments independently
 
+        if hasattr(experiments, "detectors"):
+            detectors = experiments.detectors()
+        else:
+            detectors = list(set(expt.detector for expt in experiments))
+        try:
+            test_subcache = self.detector_panel_sels[detectors[0]]
```

Comment:
Does `test_` here mean test in any way, or is this a (cached) selection?

If latter suggest sel_(name) is better?